src/pages/api/bloglist.tsx

```diff
@@ -0,0 +1,38 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import getFormattedDate from '@/lib/getFormattedDate';
+
+// blog post list
+const BlogList = (blogPosts: any[]) => `
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>Blog List</title>
+      </head>
+      <body>
+        <h1>Blog Page!</h1>
+        <ul>
+          ${blogPosts.map((post) => `<li>${post.title}</li>`).join('')}
+        </ul>
+      </body>
+    </html>
+  `;
```

Comment:
instead of returning plain HTML you should work on creating a react component and put it in the pages folder and next will automatically serve it for you

Reply:
Done ☑️